config/parse.go

```diff
@@ -316,7 +316,7 @@ func resolveHandlers(c *Config) error {
 			logger = logger.WithField("transformer", t.Name)
 			logger.Debug("Using predefined transformer")
 
-			h.Handler.Transformers = append(h.Handler.Transformers, t.T)
+			h.Handler.Transformers = append(h.Handler.Transformers, c.Transformers[t.Name].Transformer)
```

Comment:
This isn't really a fix.

resolveHandlers() run after resolveTransformers(). There are no valid scenarios where t.T should be unset at this stage if c.Transformers[t.Name].Transformer exists. Setting that is the sole purpose of resolveTransformers() ... so why is this happening in the first place? It means a TransformerRef has been corrupted, which is a much bigger problem than just this specific issue.

hmmm.... soo.....

After about an hour, I think I found the issue.... new commit will sort of explain, but it opens more questions as well.